ibm_catalog.json

```diff
@@ -0,0 +1,460 @@
+              },
+              {
+                "key": "cloud_logs_retention_period",
+                "options": [
+                  {
+                    "displayname": "7",
+                    "value": 7
+                  },
+                  {
+                    "displayname": "14",
+                    "value": 14
+                  },
+                  {
+                    "displayname": "30",
+                    "value": 30
+                  },
+                  {
+                    "displayname": "60",
+                    "value": 60
+                  },
+                  {
+                    "displayname": "90",
+                    "value": 90
+                  }
+                ]
+              },
+              {
+                "key": "logs_cos_bucket_name"
+              },
+              {
+                "key": "metrics_cos_bucket_name"
+              },
+              {
+                "key": "cloud_logs_cos_buckets_class",
+                "options": [
+                  {
+                    "displayname": "Standard",
+                    "value": "standard"
+                  },
+                  {
+                    "displayname": "Vault",
+                    "value": "vault"
+                  },
+                  {
+                    "displayname": "Cold",
+                    "value": "cold"
+                  },
+                  {
+                    "displayname": "Smart",
+                    "value": "smart"
+                  },
+                  {
+                    "displayname": "One-Rate Active",
+                    "value": "onerate_active"
+                  }
+                ]
+              },
+              {
+                "key": "skip_cos_kms_iam_auth_policy"
+              },
+              {
+                "key": "existing_kms_key_crn"
+              },
+              {
+                "key": "cloud_log_storage_key_ring"
+              },
+              {
+                "key": "cloud_log_storage_key"
+              },
+              {
+                "key": "ibmcloud_kms_api_key"
+              },
+              {
+                "key": "existing_event_notifications_instances"
+              },
+              {
+                "key": "skip_logs_routing_auth_policy"
+              },
+              {
+                "key": "logs_routing_tenant_regions"
+              },
+              {
+                "key": "logs_policies"
+              }
+            ],
+            "iam_permissions": [
+              {
+                "service_name": "logs",
+                "role_crns": [
+                    "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+                ]
+              },
+              {
+                "service_name": "cloud-object-storage",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+                ]
+              },
+              {
+                "service_name": "kms",
```

Comment:
Your missing HPCS permissions (since user could use HPCS instead of Key Protect). There is now support to add `notes` beside each permissions, so i would use that for this use case. for reference, see https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/blob/efadbb77db8eaab392ea72ec27cef29448f88283/ibm_catalog.json#L320-L329